runtime/Makefile

```diff
@@ -10,11 +10,12 @@ QIR_STDLIB_INCLUDES_DIR?=$(MK_DIR)/qir-stdlib/target/release/build/include/
 CODE_COVERAGE?=OFF
 BUILD_TYPE?=Release
 ENABLE_WARNINGS?=ON
-ENABLE_OPENMP?=ON
 ENABLE_LIGHTNING_KOKKOS?=OFF
 ENABLE_OPENQASM?=OFF
 LIGHTNING_GIT_TAG_VALUE?=latest_release
 LIGHTNING_KOKKOS_GIT_TAG_VALUE?=latest_release
+CMAKE_ARGS ?= ""
```

Comment:
Do the spaces matter here, or is that fine?

Reply:
It worked with spaces. But will remove spaces to make it consistent with other vars. Thanks!